qemu/tests/qemu_guest_agent.py

```diff
@@ -339,9 +339,17 @@ def setup(self, test, params, env):
                 logging.info("qemu-ga is not installed or need to update.")
                 self.gagent_install(session, self.vm)
 
+            error_context.context("Check qga service running status",
+                                  logging.info)
             if self._check_ga_service(
                     session, params.get("gagent_status_cmd")):
-                logging.info("qemu-ga service is already running.")
+                output = session.cmd_output(params["cmd_check_qgaservice"],
+                                            timeout=5)
+                if output and "qemu-guest-agent" in output:
```

Comment:
This is because not only qga will have this situation, but other services will also have this situation, but the qga test cannot be interrupted because of their error message.

Reply:
Got it. thanks.